runelite-client/src/main/java/net/runelite/client/plugins/microbot/fishing/minnows/MinnowsScript.java

```diff
@@ -0,0 +1,69 @@
+package net.runelite.client.plugins.microbot.fishing.minnows;
+
+import net.runelite.api.NPC;
+import net.runelite.api.coords.WorldArea;
+import net.runelite.api.coords.WorldPoint;
+import net.runelite.client.plugins.microbot.Microbot;
+import net.runelite.client.plugins.microbot.Script;
+import net.runelite.client.plugins.microbot.util.npc.Rs2Npc;
+import net.runelite.client.plugins.microbot.util.player.Rs2Player;
+
+import java.util.concurrent.TimeUnit;
+
+public class MinnowsScript extends Script {
+    public static final WorldArea MINNOWS_PLATFORM = new WorldArea(new WorldPoint(2607, 3440, 0), 2622 - 2607, 3446 - 3440);
+    private static final int FLYING_FISH_GRAPHIC_ID = 1387;
+    private static final int FISHING_SPOT_1_ID = 7732;
+    private static final int FISHING_SPOT_2_ID = 7733;
+    public static double version = 1.0;
+    private static int TARGET_SPOT_ID = FISHING_SPOT_1_ID;
+    private NPC fishingspot;
+    private int timeout;
+
+    public boolean run() {
+        mainScheduledFuture = scheduledExecutorService.scheduleWithFixedDelay(() -> {
+            try {
+                if (!Microbot.isLoggedIn()) return;
+                if (!super.run()) return;
+                if (!MINNOWS_PLATFORM.contains(Rs2Player.getWorldLocation())) return;
```

Comment:
Try to give the user a microbot.showMessage with a sleep to understand why the script is not going further.